### ISSUE TITLE
`prefer-includes` should not trigger when `fromIndex` is present

### ISSUE THREAD

**Issue by silverwind** [sindresorhus/eslint-plugin-unicorn#689]
[Array.prototype.indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) takes a rarely used second argument `fromIndex`. If it is present, the rule should not trigger because such a case can not be replaced with `Array.prototype.includes`:

``` js
console.log(['1', '2'].indexOf('1', 1));
// => -1
```

**Comment by silverwind** [comment]
Actually, [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) also features the argument, so this is invalid.

**Comment by fisker** [comment]
This was not right before, but fixed in https://github.com/sindresorhus/eslint-plugin-unicorn/pull/411
